opal/mca/btl/usnic/btl_usnic_component.c

```diff
@@ -630,7 +630,8 @@ static mca_btl_base_module_t** usnic_component_init(int* num_btl_modules,
     hints.ep_attr = &ep_attr;
     hints.fabric_attr = &fabric_attr;
 
-    ret = fi_getinfo(FI_VERSION(1, 0), NULL, 0, 0, &hints, &info_list);
+    ret = fi_getinfo(FI_VERSION(FI_MAJOR_VERSION, FI_MINOR_VERSION),
```

Comment:
...although thinking about this more, I think you're on to something.

If we compile OMPI against libfabric v1.0.0 (i.e., API v1.0) and run with libfabric v1.1.0 (i.e., API v1.1), our compromise was that the usnic libfabric provider would say "I'm v1.1, but you(the app) are compiled against v1.0, so you're going to do MSG_PREFIX wrong -- so I'll turn it off."

But OMPI correctly handles MSG_PREFIX regardless of which way the usnic provider does it.

So we should lie here; OMPI should pass a minimum of `FI_VERSION(1,1)` to `fi_getinfo()`.

New patch coming shortly...